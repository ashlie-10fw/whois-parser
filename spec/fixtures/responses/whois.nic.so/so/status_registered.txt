Domain Name: google.so
Registry Domain ID: 34544-sonic
Registry WHOIS Server: whois.nic.so
Registrar URL: 
Updated Date: 2022-12-23T09:26:14.78Z
Creation Date: 2011-01-24T00:00:00.0Z
Registry Expiry Date: 2024-01-24T00:00:00.0Z
Registrar Registration Expiration Date: 2024-01-24T00:00:00.0Z
Registrar: Mark Monitor
Registrar Abuse Contact Email: ccops@markmonitor.com
Registrar Abuse Contact Phone: +1.2083895740
Domain Status: clientTransferProhibited https://icann.org/epp#clientTransferProhibited
Domain Status: clientUpdateProhibited https://icann.org/epp#clientUpdateProhibited
Domain Status: clientDeleteProhibited https://icann.org/epp#clientDeleteProhibited
Registry Registrant ID: CAKlg-SfRgX
Registrant Name: Domain Administrator
Registrant Organization: Google LLC
Registrant Street: 1600 Amphitheatre Parkway
Registrant City: Mountain View
Registrant State/Province: CA
Registrant Postal Code: 94043
Registrant Country: US
Registrant Phone: +1.6502530000
Registrant Fax: +1.6502530001
Registrant Email: dns-admin@google.com
Registry Admin ID: ns66c-kLSXy
Admin Name: Domain Administrator
Admin Organization: Google LLC
Admin Street: 1600 Amphitheatre Parkway
Admin City: Mountain View
Admin State/Province: CA
Admin Postal Code: 94043
Admin Country: US
Admin Phone: +1.6502530000
Admin Fax: +1.6502530001
Admin Email: dns-admin@google.com
Registry Tech ID: J8ANM-4ZMch
Tech Name: Domain Administrator
Tech Organization: Google LLC
Tech Street: 1600 Amphitheatre Parkway
Tech City: Mountain View
Tech State/Province: CA
Tech Postal Code: 94043
Tech Country: US
Tech Phone: +1.6502530000
Tech Fax: +1.6502530001
Tech Email: dns-admin@google.com
Name Server: ns1.google.com
Name Server: ns2.google.com
Name Server: ns3.google.com
Name Server: ns4.google.com
DNSSEC: unsigned
>>> Last update of WHOIS database: 2023-01-16T18:45:24.287Z <<<

For more information on Whois status codes, please visit https://icann.org/epp

TERMS OF USE: You are not authorized to access or query our WHOIS database through the use of electronic processes that are high-volume and automated.  This WHOIS database is provided by as a service to the internet community.

The data is for information purposes only. We do not guarantee its accuracy. By submitting a WHOIS query, you agree to abide by the following terms of use: You agree that you may use this Data only for lawful purposes and that under no circumstances will you use this Data to: (1) allow, enable, or otherwise support the transmission of mass unsolicited, commercial advertising or solicitations via e-mail, telephone, or facsimile; or (2) enable high volume, automated, electronic processes. The compilation, repackaging, dissemination or other use of this Data is expressly prohibited.
