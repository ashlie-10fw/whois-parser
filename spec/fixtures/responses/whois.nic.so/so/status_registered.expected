#domain
  %s == "google.so"

#domain_id
  %s == "34544-sonic"


#status
  %s == :registered

#available?
  %s == false

#registered?
  %s == true


#created_on
  %s %CLASS{time}
  %s %TIME{2011-01-24 00:00:00 UTC}

#updated_on
  %s %CLASS{time}
  %s %TIME{2022-12-23 09:26:14 UTC}

#expires_on
  %s %CLASS{time}
  %s %TIME{2024-01-24 00:00:00 UTC}

#registrar
  %s %CLASS{registrar}
  %s.name         == "Mark Monitor"

#registrant_contacts
  %s %CLASS{array}
  %s %SIZE{1}
  %s[0] %CLASS{contact}
  %s[0].type          == Whois::Parser::Contact::TYPE_REGISTRANT
  %s[0].id            == "CAKlg-SfRgX"
  %s[0].name          == "Domain Administrator"
  %s[0].organization  == "Google LLC"
  %s[0].address       == "1600 Amphitheatre Parkway"
  %s[0].city          == "Mountain View"
  %s[0].zip           == "94043"
  %s[0].state         == "CA"
  %s[0].country       == nil
  %s[0].country_code  == "US"
  %s[0].phone         == "+1.6502530000"
  %s[0].fax           == "+1.6502530001"
  %s[0].email         == "dns-admin@google.com"

#admin_contacts
  %s %CLASS{array}
  %s %SIZE{1}
  %s[0] %CLASS{contact}
  %s[0].type          == Whois::Parser::Contact::TYPE_ADMINISTRATIVE
  %s[0].id            == "ns66c-kLSXy"
  %s[0].name          == "Domain Administrator"
  %s[0].organization  == "Google LLC"
  %s[0].address       == "1600 Amphitheatre Parkway"
  %s[0].city          == "Mountain View"
  %s[0].zip           == "94043"
  %s[0].state         == "CA"
  %s[0].country       == nil
  %s[0].country_code  == "US"
  %s[0].phone         == "+1.6502530000"
  %s[0].fax           == "+1.6502530001"
  %s[0].email         == "dns-admin@google.com"

#technical_contacts
  %s[0].id            == "J8ANM-4ZMch"
  %s[0].name          == "Domain Administrator"
  %s[0].organization  == "Google LLC"
  %s[0].address       == "1600 Amphitheatre Parkway"
  %s[0].city          == "Mountain View"
  %s[0].zip           == "94043"
  %s[0].state         == "CA"
  %s[0].country       == nil
  %s[0].country_code  == "US"
  %s[0].phone         == "+1.6502530000"
  %s[0].fax           == "+1.6502530001"
  %s[0].email         == "dns-admin@google.com"

#nameservers
  %s %CLASS{array}
  %s %SIZE{4}
  %s[0] %CLASS{nameserver}
  %s[0].name == "ns1.google.com"
  %s[1] %CLASS{nameserver}
  %s[1].name == "ns2.google.com"
  %s[2] %CLASS{nameserver}
  %s[2].name == "ns3.google.com"
  %s[3] %CLASS{nameserver}
  %s[3].name == "ns4.google.com"
